nss-day-cohort-39/hello-world-the-globtrotters
# Greece Automation

### Changes
- created new javascript files for automation
- automated Cities, Leaders and Landmarks List

### Testing
- make sure all information populates correctly in list
- make sure changes didn't cause unexpected errors anywhere else on the site
